roles/confluent.variables/vars/main.yml

```diff
@@ -91,8 +89,8 @@ kafka_broker:
   client_config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/client.properties"
   zookeeper_tls_client_config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/zookeeper-tls-client.properties"
   systemd_file: "{{systemd_base_dir}}/{{kafka_broker_service_name}}.service"
-  systemd_override: /etc/systemd/system/{{kafka_broker_service_name}}.service.d/override.conf
-  log4j_file: "{% if installation_method == 'archive' %}{{archive_destination_path}}/confluent-{{confluent_package_version}}{% endif %}/etc/kafka/log4j.properties"
+  systemd_override: "{{systemd_override_dir}}/{{kafka_broker_service_name}}.service.d/override.conf"
+  log4j_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/log4j.properties"
```

Comment:
although this change is fine, i wouldnt touch the log4j_file var for consistency